prisma/schema.prisma

```diff
@@ -11,14 +11,111 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Log {
-  id      String @id @default(uuid())
-  level   Level
-  message String
+model Ship {
+  id      String  @id @default(uuid())
+  name    String  @unique
+  // maneuvers Maneuver[] // I don't know how we want to handle custom types here, because what I want is a {ManeueverType, Difficulty} tuple, e.g. {"5K", "Red"}
+  // If we ever hit the case of a chassis having different hull/shields (1.0 YT-1300), we'd have to move these to the pilot
+  hull    Int
+  shields Int
+  // arcs Arc[] // Same as maneuevers - I want tuples and I don't know the right way to do it with typing, e.g. [{"front", "2'}, {"bullseye", "3'}]
+  pilots  Pilot[]
 }
 
-enum Level {
-  Info
-  Warn
-  Error
+model ShipAbility {
+  id    String  @id @default(uuid())
+  name  String
+  text  String
+  Pilot Pilot[]
+}
+
+model Pilot {
+  id             String        @id @default(uuid())
+  faction        String
+  shipId         String
+  ship           Ship          @relation(fields: [shipId], references: [id])
+  name           String
+  subtitle       String
+  initiative     Int
+  force          Int // XWS has this as a bag with the force and forceRecover packed... I don't have a gut sense of what I like better but split out for stronger typing for now
+  forceRecover   Int
+  charges        Int // Same as force, packed in XWS
+  chargesRecover Int
+  limit          Int // Fighting my inner pedant here - more technically correct feels like a "limited" boolean and then a nullable "limit" int, but a limit of 0 accomplishes the same
```

Comment:
I think you're right and the nagging feeling is correct. Maybe a rename to something like limitCount or something fixes it? If I couldn't see datatypes, I would be unsure if limit was a bool or int.

Reply:
limitCount is better 👍